mysql/datadog_checks/mysql/activity.py

```diff
@@ -205,8 +205,11 @@ def _obfuscate_and_sanitize_row(self, row):
         try:
             self._finalize_row(row, obfuscate_sql_with_metadata(row["sql_text"], self._obfuscator_options))
         except Exception as e:
+            if self._config.log_unobfuscated_queries:
+                self._log.warning("Failed to obfuscate query '%s' | err=[%s]", row["sql_text"], e)
+            else:
+                self._log.debug("Failed to obfuscate | err=[%s]", e)
```

Comment:
(not blocking) minor nit; It looks like the formatting of these logs aren't consistent in this integration and differs from Postgres and potentially SQL Server (haven't seen that PR).
- i.e. `Failed to obfuscate <> '%s': %s` or `Failed to obfuscate <> '%s' | err=[%s]`

Perhaps we can keep it consistent, I do like what you have on this line: `'%s' | err=[%s]`

Maybe something like `Failed to obfuscate <TYPE>=[%s] | err=[%s]`
- Where `TYPE` says 'query' or 'plan' given that it's appropriate to show

Reply:
By consistent, were you thinking of keeping it consistent between all the MySQL logs or between the logs in all three integrations?

Reply:
Ideally all three, that way we can fuzzy search for common attributes within the log which may prove helpful during investigation.